Eevee.Sleep.Bot/Modules/SlashCommands/RoleManagementSlashModule.cs

```diff
@@ -0,0 +1,138 @@
+using Discord;
+using Discord.Interactions;
+using JetBrains.Annotations;
+using Eevee.Sleep.Bot.Controllers.Mongo;
+using Eevee.Sleep.Bot.Utils;
+using Eevee.Sleep.Bot.Enums;
+using Discord.WebSocket;
+
+namespace Eevee.Sleep.Bot.Modules.SlashCommands;
+
+[Group("role", "Commands for managing the roles on a user.")]
+public class RoleManagementSlashModule : InteractionModuleBase<SocketInteractionContext> {
+    private async Task SendEphemeralMessageToBeDeletedAsync(string text, MessageComponent components) {
+        await Context.Interaction.RespondAsync(
+            text: text, 
+            components: components, 
+            ephemeral: true
+        );
+        
+        // Ephemeral messages cannot be deleted by the button handler, so the message containing the button should be deleted after 30 seconds.
+        await Task.Delay(TimeSpan.FromSeconds(30));
+        await Context.Interaction.DeleteOriginalResponseAsync();
+    }
+
+    [SlashCommand("display", "Select which of the role you have to display.")]
```

Comment:
"Select the role to display."

Also add a message somewhere in the selection interaction indicating that "All tracked roles will be removed from after selecting the role to display. Role ownership won't get affected by the removal, only the role assignment on Discord is affected."